.github/workflows/config/spelling_allowlist.txt

```diff
@@ -164,6 +171,7 @@ discoverable
 discretization
 discretize
 discretized
+ecore
```

Comment:
I don't think this is a word and therefore should probably not be added to the allowlist.